tests/densify/test_mvs_utils.py

```diff
@@ -0,0 +1,42 @@
+"""Unit tests for utilities for MVS methods
+
+Authors: Ren Liu
+"""
+import unittest
+
+import numpy as np
+from scipy.spatial.transform import Rotation as R
+
+import gtsfm.densify.mvs_utils as mvs_utils
+
+
+class TestMVSUtils(unittest.TestCase):
+    """Unit tests for utilities for MVS methods."""
+
+    def test_piecewise_gaussian_below_threshold(self) -> None:
+        """Unit test for the case that the angle between two coordinates is below the threshold,
+        where sigma_1 is used to calculate the score"""
+
+        xPa = np.array([15.0, 7.5, 0.0])
+        bRa = R.from_euler("z", 4, degrees=True)
+        xPb = bRa.apply(xPa)
+
+        score = mvs_utils.piecewise_gaussian(xPa=xPa, xPb=xPb, theta_0=5, sigma_1=1, sigma_2=10)
+
+        self.assertAlmostEqual(score, np.exp(-(1.0 ** 2) / (2 * 1.0 ** 2)))
+
+    def test_piecewise_gaussian_above_threshold(self) -> None:
+        """Unit test for the case that the angle between two coordinates is above the threshold,
+        where sigma_2 is used to calculate the score"""
+
+        xPa = np.array([0.0, 12.0, 15.0])
+        bRa = R.from_euler("x", 10, degrees=True)
```

Comment:
```python
bRa = Rot3.RzRyRx(rx=np.deg2rad(10), ry=0, rz=0)
bRa.transformFrom(xPa)
```

Reply:
too hard to reason about the rays. if we just use Ayush's min triangulation thing, we dont need this -- just test the gaussian values

Reply:
resolved